diff --git a/calendar_view.module b/calendar_view.module
index 647f19d..2cf7c2f 100644
--- a/calendar_view.module
+++ b/calendar_view.module
@@ -176,9 +176,14 @@ function template_preprocess_calendar_view_pager(array &$variables) {
   $current_label = $date_formatter->format($current, $date_format, $date_pattern);
   $next_label = $date_formatter->format($next, $date_format, $date_pattern);
 
+//  if ($parameters['use_previous_next'] ?? FALSE) {
+//    $previous_label = '';
+//    $next_label = '';
+//  }
+
   if ($parameters['use_previous_next'] ?? FALSE) {
-    $previous_label = t('Previous');
-    $next_label = t('Next');
+    $previous_label = '';
+    $next_label = '';
   }
 
   // Build navigation.
@@ -198,6 +203,7 @@ function template_preprocess_calendar_view_pager(array &$variables) {
           'F Y'
         ),
       ]),
+      'class' => 'text',
     ],
     // @todo limit in past?
     '#access' => TRUE,
@@ -207,8 +213,9 @@ function template_preprocess_calendar_view_pager(array &$variables) {
     '#type' => 'item',
     '#markup' => $current_label,
     '#attributes' => [
+//      'aria-label' => t('Current @date', [
       'aria-label' => t('Current @date', [
-        '@date' => $date_formatter->format($current, 'custom', 'F Y'),
+        '@date' => $date_formatter->format($current, 'custom', 'F'),
       ]),
     ],
   ];
@@ -225,7 +232,7 @@ function template_preprocess_calendar_view_pager(array &$variables) {
 
       $variables['items']['reset'] = [
         '#type' => 'link',
-        '#title' => t('Reset'),
+        '#title' => t('Сьогоднi'),
         '#url' => Url::fromRoute($route_name, [], [
           'query' => ['calendar_timestamp' => 'today'] + $parameters,
         ]),
@@ -252,6 +259,7 @@ function template_preprocess_calendar_view_pager(array &$variables) {
           'F Y'
         ),
       ]),
+      'class' => 'text',
     ],
     // @todo limit in future?
     '#access' => TRUE,
